src/mavlink_receive.c

```diff
@@ -100,6 +100,27 @@ static void mavlink_handle_msg_timesync(const mavlink_message_t *const msg)
   }
 }
 
+static void mavlink_handle_msg_small_imu_hil(const mavlink_message_t * const msg)
```

Comment:
Is a `SMALL_IMU_HIL` message any different from a `SMALL_IMU` message? If not, do we want to just use the existing message type, or did you have future changes in mind?